### PR TITLE
Fix test-runner image: use python2 for bootstrap

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

The k8s/test-infra bootstrap python script doesn't work on Python 3,
so let's force it to use Python 2.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/images/test-runner/Dockerfile [hunk 1]
```diff
@@ -35,6 +35,10 @@ RUN apt update && apt install -y --no-install-recommends \
     openssh-client \
     pkg-config \
     procps \
+    python2 \
```
Comment (afrittoli) [inline tekton/images/test-runner/Dockerfile:38]: python2 is already installed on the image, but this doesn't harm

**Comment by afrittoli** [comment]
let's wait for Mario to report

File: tekton/images/test-runner/Dockerfile (hunk 1 above)
Comment (vdemeester) [reply tekton/images/test-runner/Dockerfile:38]: It is not :stuck_out_tongue: (except if it's already install on the base debian image which I doubt)

**Comment by afrittoli** [comment]
Maybe Mario is not alive after I upgraded the cluster yesterday, let me check...

**Comment by afrittoli** [comment]
https://dashboard.dogfooding.tekton.dev/#/namespaces/mario/taskruns/build-and-push-wn77t-65c86

**Comment by vdemeester** [comment]
:pray:

**Comment by afrittoli** [comment]
Running: https://dashboard.dogfooding.tekton.dev/#/namespaces/mario/taskruns/build-and-push-mjxrg-n9mjj - hopefully the update will work this time.

**Pull request merged**